ko/news/_posts/2018-10-18-ruby-2-5-3-released.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "루비 2.5.3 릴리스"
+author: "nagachika"
+translator: "shia"
+date: 2018-10-18 12:30:00 +0000
+lang: ko
+---
+
+루비 2.5.3이 릴리스 되었습니다.
+
+2.5.2 패키지에는 빌드에 필요한 몇몇 파일이 빠져있었습니다.
+
```

Comment:
English article doesn't have this empty line.

Reply:
fixed 7b7d90ab